audit/audit.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -x -e
+CNCF_GCP_ORG=758905017065
+
+# gcloud organizations describe $CNCF_GCP_ORG 2>&1
+# ERROR: (gcloud.organizations.describe)
+# User [hh@ii.coop] does not have permission to access organization []
+
+for format in json yaml
+do
+    gcloud organizations get-iam-policy $CNCF_GCP_ORG --format=$format \
+           > cncf-org-policy.$format
+    gcloud projects get-iam-policy kubernetes-public --format=$format \
```

Comment:
Rather than "kubernetes-public" shouldn't we be doing a loop over all projects?

e.g.

```
gcloud projects list \
    --filter "parent.id=758905017065" \
    --format "value(name, projectNumber)" \
    | while read NAME NUM; do \
        gcloud projects get-iam-policy $NAME --format=$format > $NAME.policy.$format
        gcloud iam roles list --project=$NAME --format=$format > $NAME.roles.$format
    done
````

Reply:
Also, I vote for JSON - much easier to visually verify

Reply:
Currently doing json and yaml, I can remove yaml if you like.